codeworld-server/src/Main.hs

```diff
@@ -253,19 +254,19 @@ deleteProjectHandler = private $ \userId ctx -> do
             projectFile projectId
     liftIO $ removeFileIfExists file
 
-listFolderHandler :: CodeWorldHandler
```

Comment:
Can you leave the `listFolder` endpoint in place at first?  I can delete it later.  This way, I can update the server without breaking clients that have old JavaScript.

Reply:
ok